wpilibc/Athena/src/PWM.cpp

```diff
@@ -43,7 +43,7 @@ PWM::PWM(uint32_t channel) {
 
   m_channel = channel;
 
-  setPWM(m_pwm_ports[m_channel], kPwmDisabled, &status);
+  setPWMRaw(m_pwm_ports[m_channel], kPwmDisabled, &status);
```

Comment:
In C++, SensorBase grabs arrays of all ports at robot boot, and just uses those instead of grabbing new ones. That will be changed with the port resource changes later.